app/src/main/res/layout/revision_card_fragment.xml

```diff
@@ -164,6 +165,7 @@
               android:paddingEnd="@dimen/topic_revision_summary_view_subtopic_title_text_view_padding_end"
               android:paddingBottom="16dp"
               android:text="@{viewModel.computeTitleText(viewModel.nextSubtopicLiveData)}"
+              android:contentDescription="@{viewModel.computeContentDescriptionText(viewModel.nextSubtopicLiveData)}"
```

Comment:
You will need a test for this in `RevisionCardFragmentTest`  like below:

```
  @Test
  fun testRevisionCard_previousSubtopicTitle_hasCorrectContentDescription() {
    launch<RevisionCardActivity>(
      createRevisionCardActivityIntent(
        context,
        profileId.internalId,
        "test_topic_id_0",
        subtopicId = 1,
        FRACTIONS_SUBTOPIC_LIST_SIZE
      )
    ).use {
      testCoroutineDispatchers.runCurrent()
      onView(withId(R.id.prev_subtopic_title)).check(matches(ViewMatchers.withContentDescription("Replace this with the strin you expect to be read out by talkback")))
    }
  }
```
Please be sure to replicate the test for the "next" navigation card.

Reply:
@adhiamboperes I have added 2 more tests to test the behavior of the above video in the description, PTAL.